src/config.c

```diff
@@ -75,6 +75,9 @@ void load_config(struct main_window *w)
 	}
 
 	CONFIG_FIELDS(LOAD);
+
+	w->audioInputStr    = g_key_file_get_string(w->config_file, "tg", "audioInputStr", 	   NULL);
+	w->audioInterfaceStr= g_key_file_get_string(w->config_file, "tg", "audioInterfaceStr",  NULL);
```

Comment:
If the config file doesn't have this key, then `w->audioInterfaceStr` is NULL and the audio dialog will crash.